spec/requests/api/blueprints_spec.rb

```diff
@@ -328,6 +329,9 @@
         :action   => "edit",
         :resource => {
           :bundle => {
+            :service_dialog  => {
```

Comment:
@imtayadeway added this in because it is a test that catches the exact problem that was reported by @dtaylor113. However, it's pretty trivial, so I can remove this test.